holidays/calendars/chinese.py

```diff
@@ -1307,6 +1307,24 @@ class _ChineseLunisolar:
         2053: (FEB, 18),
     }
 
+    WINTER_SOLSTICE_THRESHOLDS: dict[str, dict[str, dict[int, int]]] = {
+        # UTC+7.
+        VIETNAMESE_CALENDAR: {
+            "dec23_thresholds": {3: 1943},
+            "dec21_thresholds": {0: 1980, 1: 2017, 2: 2050, 3: 2083},
```

Comment:
The parent container already has _THRESHOLDS suffix, do we need it in these names too?